lib/modules/datasource/jenkins-plugins/readme.md

```diff
@@ -0,0 +1,5 @@
+The `jenkins-plugins` datasource supports lookups from the [Jenkins Update Center](https://updates.jenkins.io/) or custom registries.
+
+Custom registries defined via `registryUrl` are expected to follow the [update site layout](https://github.com/jenkins-infra/update-center2/blob/master/site/LAYOUT.md).
```

Comment:
```suggestion
When you define a custom registry with the `registryUrl` config option, you must follow [Jenkins' update site layout](https://github.com/jenkins-infra/update-center2/blob/master/site/LAYOUT.md).
```

We should use the word `must` to indicate things that you must do, or Renovate must have to work.

Also, we should say _who_ does _what_. I think in this example the user would set a custom registry, and then must follow some rules from Jenkin's for Renovate to work.

I've also updated the link name so it's clear where you'll end up.